content/report_RHF.tex

```diff
@@ -1,9 +1,249 @@
+that have $\alpha$ as their spin component, we can write a Hartree-Fock equation and
+then simply integrate the spin component out by adding $\bra{\alpha}$ on every
+side of the equation. For the one electron part and the Coulomb operators, there is
+no problem at all, since all electrons stay in their respective orbitals. However,
+the exchange operator will move electrons around to different spin-orbitals.(see 
+Equation \ref{eq:exchange}) When the electron from an $alpha$ spin-orbital would
+be moved to a $beta$ spin-orbital, this would result in a zero, since $\bra{\alpha}\ket{\beta} = 0$.
+
+So here it is clear that the exchange operator will only remain active after spin 
+elimination between electrons that have the same spin function. So now that we have
+gotten rid of spin, all that is left is Equation \ref{eq:spinelim}.
+
+\begin{equation}\label{eq:spinelim}
+    \hat{f}^{\alpha}(1)\psi^{\alpha}_i = \epsilon^{\alpha}_i\psi^{\alpha}_i
+\end{equation}
+Off course the same argument stands for the $\beta$-spins.
+
+Now we have come to a point where we can define a restricted closed shell system.
+Imagine what would happen if all orbitals were doubly occupied, with an $\alpha$
+and a $\beta$ electron, all according to the Pauli princple. This has some consequences.
+The first one is that the amount of $\alpha$-electrons would equal the amount of
+$\beta$-electrons. Secondly, this would mean that the difference between these electrons
+is only their spin function. They reside in the same orbitals. This means that we only
+need to account for half the electrons, since the other half is in the same orbitals,
+just with a different spin. Now think about the Fock operator. Let us
+take a look at a single electron. It has a one election hamiltonian, electronic repulsion
+for all other electrons in the system and it only "exchanges" with half of the other
+electrons, only those with the same spin. This means we have double the amount of 
+
+Coulomb operators to exchange operators. When we now look back at Equation \ref{eq: fock matrix},
+we see that we are finally there.
+
+\subsection{Application of the Hartree-Fock Theory}
+\label{subsec:applic}
+At this point, we might think back to the point where we defined the Coulomb and
+exchange operators. These operators themselves depend on the orbitals. So we would
+need to know the orbitals to define our operator. It seems we are stuck again.
+However we can apply some mathemathical tricks to get us out of this impasse. 
+First we can assert that every function can be expressed as a linear combination
+of basis functions. We also know that the eigenfunctions of a Hermitian operator
+form such a set of basis functions. We begin by selecting a convenient basis set.
+This could be a set of atomic orbitals. Then we would solve the eigenproblem in
+Equation \ref{eq:eigenproblem}.
+
+\begin{equation}\label{eq:eigenproblem}
+    \boldsymbol{\hat{H}}^{core}\boldsymbol{C} = \boldsymbol{SC}c

```

Comment:
This is more of a general comment, but each equation should have all of its symbols explained in the surrounding text.